src/main/java/org/sagebionetworks/web/client/widget/entity/annotation/AnnotationsRendererWidget.java

```diff
@@ -75,13 +71,6 @@ public Widget asWidget() {
 
 	@Override
 	public void onEdit() {
-		preflightController.checkUploadToEntity(bundle, () -> {
-			if (DisplayUtils.isInTestWebsite(cookies)) {
-				editorDialogV2.configure(bundle.getEntity().getId());
-			} else {
-				getEditAnnotationsDialog().configure(bundle);
-
-			}
-		});
+		preflightController.checkUploadToEntity(bundle, () -> getEditAnnotationsDialog().configure(bundle));
```

Comment:
Drop alpha mode behavior for showing the editor in the annotations widget. With the other changes, if you're in alpha mode, you would never even see this widget.